services/slack/intermediate.go

```diff
@@ -357,6 +357,19 @@ func addFileToPost(file *SlackFile, uploads map[string]*zip.File, post *Intermed
 	return nil
 }
 
+func (t *Transformer) CreateIntermediateUser(user string) {
```

Comment:
Can you change variable name to `userId` or `username` to specify what kind of field we are getting here.